README.md

```diff
@@ -1,13 +1,13 @@
 # Nighthawk
 
-*A L7 (HTTP/HTTPS/HTTP2) performance characterization tool*
+_A L7 (HTTP/HTTPS/HTTP2) performance characterization tool_
 
 ## Current state
 
 Nighthawk currently offers:
 
 - A load testing client which supports HTTP/1.1 and HTTP/2 over HTTP and HTTPS.
-(HTTPS certificates are not yet validated).
+  (HTTPS certificates are not yet validated).
```

Comment:
It seems we use different linting tools for markdown files, and both ways satisfy the formatting checks. It would be nice of we could somehow force-sync that up across contributors. Probably out of scope here, just an observation.

Reply:
Agreed, I'll revert these changes. I thought they had come from upstream but they were from my own editor all along! 

I would be interested to match any tools you're using.

Reply:
> Agreed, I'll revert these changes. I thought they had come from upstream but they were from my own editor all along! (or the script that generates README.md via some linter on my machine?)
> 
> I would be interested to match any tools you're using.

I'm using vscode with this extension: https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint. But imho ideally we'd have a check/fix format that would be more helpful here, apparently the current check is satisfied with both your and mine version.